### DISCUSSION TITLE
`string:: 20231009` entries added each time I mark a task DONE

### DISCUSSION THREAD

**Discussion by GRIT-Nic** [YU000jp/logseq-plugin-confirmation-done-task#40] [Q&A]
As you can see in the screenshot below, I keep getting these `string` entries added to my tasks when I mark them complete. I've confirmed by disabling other plugins and trying again that this plugin is causing this to occur. 
 
![image](https://github.com/YU000jp/logseq-plugin-confirmation-done-task/assets/102177233/5f73e6eb-a64a-449f-aa9b-2998106a3f1c)

**Comment by YU000jp** [comment]
@GPS-Nic   I had not included it in the description. The property that allow discovery in queries. It is being improved. https://github.com/YU000jp/logseq-plugin-confirmation-done-task/issues/36

**Comment by GRIT-Nic** [comment]
Gotcha, thanks!!
